codeforamerica/document-transfer-service
# CCAP-132: Microservice skeleton with basic health check.

#### 🔗 Jira ticket

CCAP-132

#### ✍️ Description

Implemented a basic microservice, using ruby-grape , that includes a basic health check, instrumentation, and tests.

#### 📷 Design reference

https://www.notion.so/cfa/Tech-Spec-Document-Transfer-Microservice-d89aa25d41e447b3920ff07ab03ce7c1?pvs=4

#### ✅ Completion tasks

To test:

* Clone branch
* Install dependencies (`bundle install`)
* Launch the service (`bundle exec rackup`)
* Try to hit the endpoint (`https://localhost:9292`)
* You should receive an HTTP 200 with a JSON body (`{"status":"ok"}`)

- [x] Meets acceptance criteria
